content/preferences-settings/miscellaneous.md

```diff
@@ -7,6 +7,9 @@ draft: false
 
 # interface
 
+show splash screen at startup
+: When activated, darktable will display a splash screen indicating progress loading and initializing its code and data and scanning for updated sidecars until it is ready to display its main window.  When disabled, darktable displays nothing until loading is complete and it can display the lighttable - or darkrooom, if invoked with a single filename on the command line (default on).
```

Comment:
I corrected this on your last PR but since you're doing a few I thought I'd mention that we don't use double-spaces after full stops in dtdocs. Also please can you use a double-dash `--` in place of the single dash above